<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightVerify_Dialog</class>
 <widget class="QDialog" name="LightVerify_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>446</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>429</width>
    <height>446</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>429</width>
    <height>446</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Light options</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>208</y>
     <width>97</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Intensity</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="_105">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>234</y>
     <width>105</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Distant light</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="_106">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>234</y>
     <width>93</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>White light</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>260</y>
     <width>95</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Display size</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>288</y>
     <width>197</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Fade parameters</string>
   </property>
   <widget class="QCheckBox" name="_108">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>24</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Enable fade</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>46</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>76</y>
      <width>65</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Exponent</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="_109">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>46</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="_110">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>76</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>288</y>
     <width>197</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Spot parameters</string>
   </property>
   <widget class="QCheckBox" name="_111">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>24</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Enable spot</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>48</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cone angle</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>78</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Concentration</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="_112">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>47</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="maximum">
     <double>180.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="_113">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>78</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="_2">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_1">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="Defaults">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>8</y>
     <width>400</width>
     <height>87</height>
    </rect>
   </property>
   <property name="title">
    <string>Defaults</string>
   </property>
   <widget class="QPushButton" name="_1002">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>55</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select Style</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../RealCAD.qrc">
      <normaloff>:/images/PROPERTIES-styles.svg</normaloff>:/images/PROPERTIES-styles.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_103">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>56</y>
      <width>37</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Weight</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_1001">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>28</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select Layer</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../RealCAD.qrc">
      <normaloff>:/images/PROPERTIES-layers.svg</normaloff>:/images/PROPERTIES-layers.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="_102">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>56</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_100">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>31</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Colour</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_102">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>57</y>
      <width>39</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Style</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_1000">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>29</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select Colour</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../RealCAD.qrc">
      <normaloff>:/images/PROPERTIES-colours.svg</normaloff>:/images/PROPERTIES-colours.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="_100">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="WeightSpinBox" name="_103">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>54</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>3.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="LayerSpinBox" name="_101">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>28</y>
      <width>149</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="maximum">
     <number>2049</number>
    </property>
   </widget>
   <widget class="QPushButton" name="_1003">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>54</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select Weight</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../RealCAD.qrc">
      <normaloff>:/images/PROPERTIES-weights.svg</normaloff>:/images/PROPERTIES-weights.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_101">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>33</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Layer</string>
    </property>
   </widget>
  </widget>
  <widget class="QDoubleSpinBox" name="_104">
   <property name="geometry">
    <rect>
     <x>208</x>
     <y>206</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="_107">
   <property name="geometry">
    <rect>
     <x>208</x>
     <y>260</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QGroupBox" name="_2000">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>100</y>
     <width>397</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Sun Light Properties</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>228</x>
      <y>30</y>
      <width>39</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>232</x>
      <y>52</y>
      <width>35</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>32</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Longitude:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>52</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Latitude:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>67</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TIme Zone:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Date">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>30</y>
      <width>85</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>yyyy/mm/dd</string>
    </property>
   </widget>
   <widget class="QLabel" name="Latitude">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>52</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>degrees</string>
    </property>
   </widget>
   <widget class="QLabel" name="Time">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>52</y>
      <width>65</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>hh:mm:ss</string>
    </property>
   </widget>
   <widget class="QLabel" name="Longitude">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>32</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>degrees</string>
    </property>
   </widget>
   <widget class="QLabel" name="TimeZone">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>00.00</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WeightSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>../RealCADQt/source/weightspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>LayerSpinBox</class>
   <extends>QSpinBox</extends>
   <header>../RealCADQt/source/layerspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_100</tabstop>
  <tabstop>_1000</tabstop>
  <tabstop>_101</tabstop>
  <tabstop>_1001</tabstop>
  <tabstop>_102</tabstop>
  <tabstop>_1002</tabstop>
  <tabstop>_103</tabstop>
  <tabstop>_1003</tabstop>
  <tabstop>_104</tabstop>
  <tabstop>_105</tabstop>
  <tabstop>_106</tabstop>
  <tabstop>_107</tabstop>
  <tabstop>_112</tabstop>
  <tabstop>_113</tabstop>
  <tabstop>_109</tabstop>
  <tabstop>_110</tabstop>
  <tabstop>_1</tabstop>
  <tabstop>_2</tabstop>
  <tabstop>_108</tabstop>
  <tabstop>_111</tabstop>
 </tabstops>
 <resources>
  <include location="../RealCAD.qrc"/>
 </resources>
 <connections/>
</ui>
